lib/l10n/app_de.arb

```diff
@@ -16,6 +16,7 @@
   "remove": "Entfernen",
   "before": "Vorher",
   "after": "Nachher",
+  "preview": "Vorschau",
```

Comment:
Maybe "Vorschau anzeigen" would be better?

Reply:
"Vorschau anzeigen" if it's not expanded, "Vorschau" if it's expanded?

Reply:
That seems weird... Why not "Vorschau verstecken"?

Reply:
You know there is a "Ausklappen" or "Einklappen" button next to that right?

Reply:
Maybe I should have tested this out :D Then nevermind